oshi-core/src/main/java/oshi/software/os/mac/MacOperatingSystem.java

```diff
@@ -107,12 +108,18 @@ public OSProcess getProcess(int pid) {
                 }
             }
         }
+        long bytesRead = 0, bytesWritten = 0;
+        RUsageInfoV2 rUsageInfoV2 = new RUsageInfoV2();
+        if (0 == SystemB.INSTANCE.proc_pid_rusage(pid, SystemB.RUSAGE_INFO_V2, rUsageInfoV2)) {
```

Comment:
Should we check for OS version >= 10.9 here?  What happens if this is run on an earlier version?  I suspect an exception (if the method doesn't exist) or possibly it populates a smaller structure and would give memory access errors/segfault on the next two lines.

We could use a utility method to find OS X version (need to check for 12+ for #260) so might as well dual-purpose that.